manifests/apache_exporter.pp

```diff
@@ -86,6 +90,8 @@
   Optional[Hash] $scrape_job_labels                          = undef,
   Optional[String[1]] $proxy_server                          = undef,
   Optional[Enum['none', 'http', 'https', 'ftp']] $proxy_type = undef,
+  Stdlib::Absolutepath $web_config_file                      = '/etc/apache_exporter_web-config.yml',
```

Comment:
I'm not a fan of polluting `/etc`. Does the prometheus community recommend this or is there any standard directory we could use?

Reply:
I am not familiar with the exporters to answer that. But for package installations we do not manage any other file resources and as far as i saw not every exporter creates the `/etc/prometheus/`. For URL installations we could use the `$extract_path` but that is only defined in the `prometheus::daemon` define and not inside the exporter class.

Reply:
@rwaffen Have you maybe additional input why we should use `/etc/<exporter name>_exporter_web-config.yml`? Since i only advanced based on the existing code base.